examples/noisy_mbqc.py

```diff
@@ -18,7 +18,7 @@
 theta = np.random.rand(2)
 circuit.rz(0, theta[0])
 circuit.rz(1, theta[1])
-circuit.cnot(0, 1)
+circuit.cnot(0,1)
```

Comment:
This line should not have been changed.

Reply:
More generally, all the changes in this file seem to be only reformatting: maybe we should reformat this file, but it should be done in another PR.

Reply:
Is it really useful to make a whole PR for reformatting? I could maybe just revert the changes implied by the wrong use of black for the concerned files?